omnibus/config/software/aerospike-py2.rb

```diff
@@ -7,7 +7,7 @@
 build do
   env = {
     "DOWNLOAD_C_CLIENT" => "0",
-    "AEROSPIKE_C_HOME" => "#{install_dir}/embedded/lib/aerospike",
+    "AEROSPIKE_C_HOME" => "#{Omnibus::Config.source_dir()}/aerospike-client/aerospike",
```

Comment:
Is `#{Omnibus::Config.source_dir()}/aerospike-client/aerospike` the same as `#{project_dir}/aerospike`? If so, the latter reads a little easier on the eye.

Reply:
It's not, because `project_dir` is not the (omnibus) project's directory but the software definition's working directory (yeah, this is strange, you can read: https://github.com/DataDog/omnibus-ruby/blob/781517d594ee5481709a6f6c5b9180b6f3dd7fa7/lib/omnibus/software.rb#L589-L592 and then 😭).
So here `project_dir = Omnibus::Config.source_dir()/aerospike-py2`, which is not what we want 😢.